docs/Home.md

```diff
@@ -101,12 +102,35 @@ await InitializeArcJs({
   });
 ```
 
-### Work with Arc Contracts
+You can use the Arc.js [ConfigService](Configuration) to set the provider host and port that web3 uses to connect your applicaton to the net when not using MetaMask.  But you can also tell `InitializeArcJs` to use the settings stored in the Arc.js `truffle.js` file for mainnet (live), kovan, ropsten and ganache.  Here is an example of telling Arc.js to use its default settings for Kovan:
+
+```javascript
+await InitializeArcJs({
+  "useNetworkDefaultsFor": "kovan"
+  });
+```
+
+!!! info
+    For safety, truffle.js specifies a different HTTP port for each network.  You will need to make sure that the testnet you are using is listening on that port.  The port values are:
+
+    <table style="margin-left:2.5rem">
```

Comment:
Is this here by mistake ?

Reply:
Do you mean the Html?  No, that is intentional

Reply:
(I want the indentation)

Reply:
markdown supports tables and indentation.

Reply:
> markdown supports tables and indentation.

I know, but not in the case where I needed it.  It is possible it would work in this case, but not in the original case from whence I copied this snippet.